### PR TITLE
PP-1267 Project bootstrapping of Pay Admin Dashboard

### PR DESCRIPTION
**Pull request by ckalista**
Bootstrapping of Pay Admin Dashboard a tool designed to help coordinating the release process, showing what release is at what stage through the pipeline as well as facilitating the approval and promotion operations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
 # Pay Admin Dashboad
+
+Pay Admin Dashboard offers a convenient way to get insight into the release pipeline of Gov.uk Pay. The process around releasing software to production entails a sequence of environments that application artifacts are deployed to and a couple of gates that ensures the quality of the deployment. An application artifact starts in TEST environment where it is integrated and tested. Before being allowed to progress it must be signed by an approver then promoted to each of the following environments in turn, STAGING and finally PRODUCTION.
```
Comment (kakumara) [inline README.md]: `GOV.UK Pay`

File: public/plugins/octokat/LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2015 Philip Schatz
```
Comment (kakumara) [inline public/plugins/octokat/LICENSE.md]: Does these `public plugins` needs to be committed? I can see whole load of library code being committed below (ex. octokat, bower etc.). This means they become mantainable artefacts in our codebase. Can we sort of do this by a build time install thing perhaps during `npm install` or something similar?

File: README.md (hunk 1 above)
Comment (ijmad) [reply README.md]: Also needs fixing in the title of the page

**Comment by kakumara** [comment]
Should we dockerise this so that it becomes one of us. (see `demo-service`) Otherwise it looks like an outcast :)
_btw, I'm not bothered about this being merged as it and do it as a separate commit._

**Comment by kakumara** [comment]
minor: almost all of your files missing the `new line` char at the end of file.

**Comment by kakumara** [comment]
Other than that good as a first pass 👍

File: public/plugins/octokat/LICENSE.md (hunk 2 above)
Comment (ckalista) [reply public/plugins/octokat/LICENSE.md]: I originally had an issue pulling this library through npm - however here is the workaround: https://github.com/webpack/webpack-dev-server/issues/66.

File: README.md (hunk 1 above)
Comment (ckalista) [reply README.md]: Fixed both in README.md and title.

**Pull request merged**